### PR TITLE
Added few Type II (Mixmaster) anonymous remailers to global.csv

### PR DESCRIPTION
**Pull request by pyllyukko**

### CODE REVIEW

**Comment by agrabeli** [comment]
Thank you @pyllyukko !

File: lists/global.csv [hunk 1]
```diff
@@ -1486,4 +1486,3 @@ https://disclose.ngo/,POLR,Political Criticism,2021-11-24,OONI,Reportedly blocke
 https://remailer.paranoici.org/,ANON,Anonymization and circumvention tools,2022-01-04,,Type II anonymous remailer
 https://fleegle.mixmin.net/,ANON,Anonymization and circumvention tools,2022-01-04,,Type II anonymous remailer
 http://redjohn.net/,ANON,Anonymization and circumvention tools,2022-01-04,,Type II anonymous remailer
-https://dizum.com/,ANON,Anonymization and circumvention tools,2022-01-04,,Type II anonymous remailer
```
Comment (pyllyukko) [inline lists/global.csv]: Why was this removed?
